vscode-extension/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "relay-lsp-client",
```

Comment:
Is this the name that gets used in the VSCode Marketplace? I suspect this should be human readable. Maybe "Relay GraphQL"? Not sure what the best name is here. Might need to think about it.

Reply:
So the flow one has some hyphenated name https://github.com/flow/flow-for-vscode/blob/main/package.json

I think they're like bundle identifiers in iOS. I'm fine with anything. I'll defer to you 
<img width="366" alt="image" src="https://user-images.githubusercontent.com/6754223/161679049-02e5d972-5591-407a-ab55-7aaccba87d55.png">

Reply:
Oh interesting. I wonder where the human readable bits come from.

Reply:
Use `displayName` in the `package.json` - https://code.visualstudio.com/api/references/extension-manifest

Reply:
Thanks @orta !